### ISSUE TITLE
Notifications 1.0.0 App Crash on Launch

### ISSUE THREAD

**Issue by GarvMaggu** [cph-cachet/flutter-plugins#161]
- Device: [.Redmi Note 7 Pro]
 - OS: [Android 10]

The app is crashing as soon as it is launched. Even tried the example app and also tried to implement it on my own app but the same result.

**Comment by thomasnilsson** [comment]
Got any logs? :)

**Comment by GarvMaggu** [comment]
> Got any logs? :)

Ohh sorry, forgot to provide, here are the logs in Debug Console:

Built build\app\outputs\flutter-apk\app-debug.apk.
Installing build\app\outputs\flutter-apk\app.apk...
E/AndroidRuntime(32122): FATAL EXCEPTION: main
E/AndroidRuntime(32122): Process: com.example.notificationlistener, PID: 32122
E/AndroidRuntime(32122): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.example.notificationlistener/com.example.notificationlistener.MainActivity}: java.lang.IllegalStateException: Not allowed to start service Intent { cmp=com.example.notificationlistener/cachet.plugins.notifications.NotificationListener }: app is in background uid UidRecord{bbeb5b9 u0a311 TPSL idle procs:1 seq(0,0,0)}
E/AndroidRuntime(32122): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
E/AndroidRuntime(32122): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
E/AndroidRuntime(32122): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
E/AndroidRuntime(32122): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
E/AndroidRuntime(32122): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
E/AndroidRuntime(32122): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
E/AndroidRuntime(32122): 	at android.os.Handler.dispatchMessage(Handler.java:107)
E/AndroidRuntime(32122): 	at android.os.Looper.loop(Looper.java:214)
E/AndroidRuntime(32122): 	at android.app.ActivityThread.main(ActivityThread.java:7356)
E/AndroidRuntime(32122): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(32122): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
E/AndroidRuntime(32122): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
E/AndroidRuntime(32122): Caused by: java.lang.IllegalStateException: Not allowed to start service Intent { cmp=com.example.notificationlistener/cachet.plugins.notifications.NotificationListener }: app is in background uid UidRecord{bbeb5b9 u0a311 TPSL idle procs:1 seq(0,0,0)}
E/AndroidRuntime(32122): 	at android.app.ContextImpl.startServiceCommon(ContextImpl.java:1616)

**Comment by thomasnilsson** [comment]
I am unable to recreate the issue. Did you add the service to the manifest, as stated in the README?

```xml
        <service android:name="cachet.plugins.notifications.NotificationListener"
                 android:label="notifications"
                 android:permission="android.permission.BIND_NOTIFICATION_LISTENER_SERVICE">
            <intent-filter>
                <action android:name="android.service.notification.NotificationListenerService" />
            </intent-filter>
        </service>
```

**Comment by thomasnilsson** [comment]
Fixed in version 1.0.1
